### PR TITLE
New school downloads page, remove downloads from meter management page and tidy up a bit

### PR DESCRIPTION
**Pull request by jamesjefferies**

### CODE REVIEW

File: app/views/management/schools/show.html.erb [hunk 1]
```diff
@@ -10,6 +10,8 @@
     <% if @school.configuration %>
       <%= link_to "Review your school's energy analysis", school_analysis_index_path(@school), class: 'btn btn-outline-dark rounded-pill font-weight-bold' %>
     <% end %>
+    <%= link_to "Download your data", school_downloads_path(@school), class: 'btn btn-outline-dark rounded-pill font-weight-bold' if can?(:read, :school_downloads) %>
```
Comment (jamesalmond) [inline app/views/management/schools/show.html.erb]: Thought ewe said this was going under the My School menu?

File: app/views/schools/downloads/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+<p><%= link_to 'Back to school dashboard', @school %></p>
+
+<p><%= link_to 'Download AMR data for all meters combined', school_meters_path(@school, format: :csv) %></p>
+
+<h2>Meters</h2>
+<p>Download the AMR data for each active meter</p>
+<ul>
+  <% @school.meters.active.each do |meter| %>
+    <li><%= link_to "#{meter.mpan_mprn} - #{meter.meter_type.to_s.humanize} - #{meter.name}", school_meter_path(@school, meter, format: "csv")  %></li>
```
Comment (jamesalmond) [inline app/views/schools/downloads/index.html.erb]: Trailing dash when the meter name is blank

File: app/controllers/schools/downloads_controller.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+module Schools
+  class DownloadsController < ApplicationController
+    include CsvDownloader
```
Comment (jamesalmond) [inline app/controllers/schools/downloads_controller.rb]: Do we need this?

Comment (jamesjefferies) [reply app/controllers/schools/downloads_controller.rb]: Nope, shouldn't be there

File: app/views/management/schools/show.html.erb (hunk 1 above)
Comment (jamesjefferies) [reply app/views/management/schools/show.html.erb]: It is, but you don't get the My School menu if you're a management type - it's on the menu too.

**Pull request merged**